spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -692,6 +738,9 @@
         )
 
         it 'logs the status received' do
+          allow(Rails).to receive(:cache).and_return(
+            ActiveSupport::Cache::RedisCacheStore.new(url: IdentityConfig.store.redis_throttle_url),
+          )
```

Comment:
we repeat this bit a lot in these specs. Would it make sense to either move it up to a top level context (have it once) or make a helper method like` use_redis_cache`?

Reply:
Yes, I'll look into this. 👍🏻